videoencoder.cpp

```diff
@@ -328,9 +368,9 @@ main (int argc, char *argv[])
 	  fflush (ogg_fp);
 	  fsls = 0;
 	}
-      //fprintf(stderr, "%s\n", input_png);
     }
 
+  printf("last frame\n");
```

Comment:
just debugging to verify the frame is received. It will end up in autoinst-log.txt